### ISSUE TITLE
Transfer Learning or Fine-Tuning a ChainerCV pre-trained model

### ISSUE THREAD

**Issue by The-Gupta** [chainer/chainercv#413]
How can I fine-tune a ChainerCV pre-trained model, say FasterRCNN?

In the paper - https://arxiv.org/pdf/1708.08169.pdf (titled "ChainerCV: a Library for Deep Learning in Computer Vision"), they say "To make training a model easier in such cases, ChainerCV provides reference implementations to train models, which can be used as a baseline to write new training code. In addition to that, **pre-trained models can be used together with the users dataset to fine-tune the model**. ChainerCV also provides set of tools for training a model including dataset loader, prediction evaluator, and visualization tools."

But, I can not find how a do it.
Could someone show using an example? Like, make pre-trained FasterRCNN identify another class (say, elephant) also?

**Comment by yuyu2172** [comment]
I hope discussion here helps.
https://github.com/chainer/chainercv/issues/391

**Comment by The-Gupta** [comment]
Thanks, @yuyu2172! 
I'm going through the conversation but it's inconvenient 'coz it's unorganized.
I'd appreciate an official ChainerCV documentation on it, which talks about prereqs. and steps properly.

**Comment by yuyu2172** [comment]
Yes. I think it is better for us to have a tutorial. 
I can only say that it is not really difficult to modify train.py to work with your data.
It should work by modifying VOC dataset and label names to your own.
